docs/api.md

```diff
@@ -467,6 +467,16 @@ Items are arrays when their corresponding `stdio` option is a [transform in obje
 
 [More info.](output.md#additional-file-descriptors)
 
+### result.ipc
+
+_Type_: `unknown[] | undefined`
```

Comment:
Maybe nicer to make it always an array and just say that it's empty if:

> the [`ipc`](#optionsipc) option is `true`. if the [`buffer`](#optionsbuffer) option is `false`.

?

Reply:
Sounds good. I was considering this as well.
I have updating the PR accordingly.